app/validators/users.js

```diff
@@ -10,17 +12,40 @@ const isPasswordLenghtvalid = password => {
 };
 
 const isPasswordAlphanumeric = password => {
-  if (regexAlphanumeric.test(password) === false) {
+  if (!regexAlphanumeric.test(password)) {
     return false;
   }
   return true;
 };
 
 const isEmailValid = email => {
-  if (regexValidEmail.test(email) === false) {
+  if (!regexValidEmail.test(email)) {
+    return false;
+  }
+  return true;
+};
+
+const isUserValid = user => {
+  const { email, password } = user;
+
+  if (!isPasswordLenghtvalid(password)) {
+    user.status = 'Password too short!';
     return false;
   }
+
+  if (!isPasswordAlphanumeric(password)) {
+    user.status = 'Password is not alphanumeric!';
+    return false;
+  }
+
+  user.password = encryptPassword(password);
```

Comment:
Why is this here? This is not a validation.

Reply:
Also research functional programming and pure functions.
In general a function shouldn't modify objects (no side effects).
For example, if you needed for some reason to compute a value to add it as an attribute you calculate that value in your function and the function caller then add that value to the user.